testlib/src/main/resources/scala/scalafmt/basic.dirty

```diff
@@ -1,4 +1,4 @@
-@ foobar("annot", {
```

Comment:
This change here is to eliminate the difference in formatting between .sc and .scala files so that test harness would still work the same.